spec/components/rails_bootstrap/badge_component_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+RSpec.describe RailsBootstrap::BadgeComponent, type: :component do 
+  context "variation attribute tests" do
```

Comment:
Can you change the description of the context block to maybe something like "render badge"?